TestCode/src/main/java/com/example/testcode/parameterized/FindSunday.java

```diff
@@ -0,0 +1,16 @@
+package com.example.testcode.parameterized;
+
+import java.time.LocalDate;
+import java.util.stream.Stream;
+
+public class FindSunday {
+
+
+    public static LocalDate findFirstSunday(LocalDate date) {
+        if (date.getDayOfWeek().getValue() == 7) {  // 날짜가 일요일인지 확인
```

Comment:
주석으로 `날짜가 일요일인지 확인` 을 작성하는 것 보다 코드를 보았을 때 직관적으로 볼 수 있다면 읽기 쉬운 코드를 작성하게 만들어주는게 좋습니다.
대표적으로 모든 "매직넘버"를 상수로 만들거나, 변수로서 이름을 지어주는 것입니다.

이런식으로 작성하면 어떨까요?
```suggestion
        if (date.getDayOfWeek() == SUNDAY) {  // 날짜가 일요일인지 확인
```